packages/build-info/src/frameworks/solid.ts

```diff
@@ -4,18 +4,18 @@ export class Solid extends BaseFramework implements Framework {
   readonly id = 'solid-js'
   name = 'SolidJS'
   npmDependencies = ['solid-js']
-  excludedNpmDependencies = ['solid-start']
+  excludedNpmDependencies = ['@solidjs/start']
```

Comment:
Yep, and I think it would be useful to leave comments for posterity
```suggestion
  excludedNpmDependencies = [
    // Used this name up to 0.3.11
    'solid-start',
    // Renamed starting at 0.4.0
    '@solidjs/start',
  ]
```